File: app/src/main/java/com/metahumanz/pacilread/model/ReadingBookStatRecord.java
package com.metahumanz.pacilread.model;

public class ReadingBookStatRecord {
    public String bookIdentity;
    public String bookTitle;
    public String bookAuthor;
    public int totalDurationSeconds;
    public int totalCharCount;
    public long updatedAt;
    public long localBookId = -1L;
    public String localCoverPath;
}

File: app/src/main/kotlin/com/metahumanz/pacilread/model/ReadingBookStatRecord.kt
package com.metahumanz.pacilread.model

class ReadingBookStatRecord {
    @JvmField var bookIdentity: String? = null
    @JvmField var bookTitle: String? = null
    @JvmField var bookAuthor: String? = null
    @JvmField var totalDurationSeconds: Int = 0
    @JvmField var totalCharCount: Int = 0
    @JvmField var updatedAt: Long = 0
    @JvmField var localBookId: Long = -1L
    @JvmField var localCoverPath: String? = null
}
